### PR TITLE
UCT/GTEST: Fix stress_iface_ops test

### PR DESCRIPTION
**Pull request by brminich**
## What
Minor fix of UCT rc test

### CODE REVIEW

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 2 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 06-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/6586a8a418ad26d16aac78d32913bb1f2fcdb4c4/_main) | :x: FAILURE
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/e2e5b60475a26f57d7e9f1709a6fddb874fc47bb/hpc-test-node-new_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/792863299ba74f0ef8e6f82a1594b737dc0452ed/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/9d07d5b8ba0aded572d33fbe849df7c9b19e0d5a/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/8a4aca4688781eeb9517548c06096bbc10748a05/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/a866326f62614e9b4f6108ef7b527220b9ec8549/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/b764b29846d61fb6a8eaa5209b3470c321e1a00c/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/ce61357ed50a1039c530cab68f70fbd096a21833/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/86849c3ccddcc106afefeb6f8e4e724e04dd087f/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/e47c4de4f3d02cf2a7462f4615efbd0a3367d3d4/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/6c2170014a1a5ca033c2bfec58c27fdfcfe09f6e/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/300f22f64dad8a598ffe4d6b57b47958b8622132/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/aa0b3e68590f6e02f5289ee19fdb09471e9b5f4c/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/2e751c433451626f061c125ec4266cf0fd1c70df/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/7e6a4621e11b7ee8826a7319b6c3644369b849e5/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/19990464add20b9b92f941c43bd5027a3315bfd4/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/cfd2dca201c2af86ebc392c7c07e28edd5f14464/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/cab4d963e0cdc01cb6596818a62c99a31718dfa6/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/2f0b8eced27cdc85811edfad8c381924a64994e4/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/65b6d431d1e08451dddf5e1eac6cedab11305674/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/19a509f11633f4ddfe69d662cf114a5889af2f82/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/f8fb6acf46290716b7d590cba297c49ad4241b07/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/4a0b306099891e179e50c6d307ec9b3f62c93e3b/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/af2ca2a2d17f26864bec7240a51878846a4b44ef/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/272498cdacf9601b9de051cba84c614c/raw/f73200a24fc104fddcdee9589b270781131ecbaa/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by dmitrygx** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 2 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 07-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/bcf67ac1208d89d86f182928e777e3b79cb649c9/_main) | :x: FAILURE
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/571ab3484e58487f636e9fdb42195b42a5ff016f/hpc-test-node-new_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/e2b84c78d3fea828f274eb1d546ff4b8e48a4ee8/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/2343576fb4aa1aa2915359ba87f6a8c27a7bed45/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/b219b25468b2a2c73b55a0055add6f114f5964fd/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/14d472eb27edeaf8c85fceeb9c7eb0e5f25cd503/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/e52da29052805d90e49e7338624b66af27f50d3e/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/1e6911786c1bf5ca7dc00d2c174a34c994e605ef/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/d1bf73d0342eb2b28a08a507a2a55df3f4813fd8/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/8f890842ab1d6f6b2a156c8e108b8402d5eb26a1/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/6e69c3e8e680c0ca522d4e6ff675991f358655af/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/c6a52441feb71c6d438ac08c2a8f8ef3c87ef35c/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/cda1e117f70c15ebfd5921de0058083c2ef89295/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/a1125861a6f07d074196031856c97d190e94f4af/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/908508e1c808209c45c9b6cca5281def71f47a58/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/b159db0e831cb597a612c6f55a59ae0bc8979166/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/3944748f9c08dfa081eae92fb2971a490926849f/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/64fb1bfe57dda33a68710a6522e67e27b471e6ec/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/55f34f1ec62f3b5e285d3e101ac6bd9876c03e7c/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/0a7c7c6147b14eec682ab261c615abf8b0a0e0c3/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/ed09e1baa55b59434ad9fd34b4f62c5105126c4b/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/2056cd18e947f1c7fdf967f1b2529a853e2e6959/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/73a4975d644affc579fb631295478f6f7e61a333/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/0f6aa5bae4c9c6454f0a164113546406adfbb9b0/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/805a265bba2ac14d29f424959d5db24b/raw/bc37ef13809661b2ad9d50ec755804c4d6ab9f2e/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 4 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 10-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/360eea7dab414e8b5b20ed0a2e9e4cd013f6b037/_main) | :x: FAILURE
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/77645b5f8a2101614c0df8906d63b1fc0315b548/hpc-test-node-gpu_W0) | :x: FAILURE
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/f67752c83656a7caaabfd23517bfc2de7187255a/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/20e56ec9c58fef04a3407872003abbb67b3d0318/hpc-test-node-new_W0) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/3d846bca0ed71fa1e4596c653f63c7d3a57b13fe/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/997dfea0075cbb919db617964d7c93694e6d8c1b/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/9b661db2ad0334dac719caf66cb4c90b84316745/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/4f500c60455c8fb753cc9f4daa33088b541223ab/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/e325ec1ce9bcab1cf461af17665cf887dd229e48/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/dc645b52abacb2129593ea5a230ba547d063cbac/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/948a0f0c195f3235740a96ea75f41b1d99c9352f/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/4a04330cb0a081cc9608102260bffbcd724b06dc/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/a0308ac35133f6b499597bdf01ae20bea3a042eb/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/dabe75c1f2a2cfc8f8090936db94bee124c94268/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/4c2fc86faecde8d5d317debf5c128849890530f9/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/e448b9b3ccb0bbcf5215ffd0101128e7b8ff9fe0/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/cb9da60d0739dda23df536feb8906389c3d11c0c/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/1bfc63365f64b1e471eb6b60221b67bd3dd69cb4/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/1b6f0fd394d65052dd2fbcb7fe367fb4eac7ebd3/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/61e210c3452b03853fa4e36b33407683e9128ee5/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/63969ddc272099300c7fc3464943bde1897c12c3/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/a6e3035bdf904e46780e49d4336f372494c0026d/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/f5f1c53317db8cdae754722187438a185e23ef53/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/e8e6d48644fb5f9c8f0405b8a45f434b6ec9ca91/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5dd57eb77d005604e5b38b8b9e58692a/raw/e7ebe7f28bbba074767a648b684917747d4736e6/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by brminich** [comment]
bot:retest

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 11-Feb-2020

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/dc90adbc5ea7b77c2c8db61c00597603466c8b98/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/e5248de18f6d838ce5e8fc1d4971833403c1a6d9/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/64c07a7debe7d0ed535c7a35b533e2675a612586/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/15d5796ebda724786f40c8e3c54638db15aa512f/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/90bde8ac615ab71b4ff9ffcf490ec954dbd3c39b/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/a289388e79e36fa9041e2538ef447d6b4b187c79/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/f0d11bbf317dc6309c7b7f4d0020b473c933cb02/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/65f14b7d65a47ae1fe1b9c7106d542764886176f/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/0ea7ab44409f8513d8c533aca09f15bc4fbdd20f/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/5a70821177c05780a124f34beb2dc08ca3c67123/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/4d9816a1db85f217fdc86c9fc3dcf0908f1adfc9/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/5234ca6f4489eb25c6b61490015163bbae191b5c/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/f258b99a3758a5d02764ef2d4725acdfd74f17b8/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/c1cc2ff2f33aa7cb7e48062fc8742b590d875837/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/0794159b8d9016700498b7fad200d4de6b4b21cc/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/08a312a14db6535542c2c12c2f8cc1694150f3f8/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/9c80444588e8eb92b3491e6d2d01f2ff3524d54e/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/77f4593e5cef4d270d538db709270e636908f30a/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/5063dbcd05adbb6fd14d680027fa357c00adfa1b/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/4919bb1214e36cea5a55dd9598ac0065149c8e3b/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/14752c6e7c85b07bb427b2129b6a7587374015d1/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/dc4b47321721f4dc7f2a3b278935f0856f30687c/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/a9fde2e11aa8d99ca2e568ab098556c99173bf6a/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/88c5b37440b0ad093c23d2a62b21515008e25218/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/2e56a84cabd05bec98804740146b7afe/raw/60511c094bb9dad5305b1df7aa244e9c79884e76/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: test/gtest/uct/ib/test_rc.cc [hunk 1]
```diff
@@ -52,12 +52,11 @@ void test_rc::test_iface_ops(int cq_len)
...
 
     UCS_TEST_GET_BUFFER_IOV(iov, iovcnt, sendbuf.ptr(), sendbuf.length(),
-                            sendbuf.memh(),
-                            m_e1->iface_attr().cap.am.max_iov);
+                            sendbuf.memh(), m_e1->iface_attr().cap.put.max_iov);
     // For _x transports several CQEs can be consumed per WQE, post less put zcopy
     // ops, so that flush would be sucessfull (otherwise flush will return
     // NO_RESOURCES and completion will not be added for it).
-    for (int i = 0; i < cq_len / 3; i++) {
+    for (int i = 0; i < cq_len / 5; i++) {
```
Comment (yosefe) [inline test/gtest/uct/ib/test_rc.cc]: why is this needed?

Comment (brminich) [reply test/gtest/uct/ib/test_rc.cc]: with dc we pass number of BBs instead of CQs (because there is no CQ counter in DC).
Since with correct `put.iov` limit we consume more BBs per WQ, the test fails, because of lack of BBs

Comment (yosefe) [reply test/gtest/uct/ib/test_rc.cc]: ok
